lib/commands/location.js

```diff
@@ -35,8 +35,11 @@ commands.setGeoLocation = async function (location) {
   }
   const args = ['-u', this.opts.udid];
   args.push(...formatLocationArg(latitude));
-  args.push(...formatLocationArg(longitude));
-  try {
+  if (latitude.startsWith('-')) {
+    args.push(...(longitude));
+  } else {
+    args.push(...formatLocationArg(longitude));
+  }  try {
```

Comment:
Please run lint on your local machine (this ought to be done on commit... not sure why the pre-commit hooks are not running for you).
```
npm run lint
```